tests/examples/test_compressed_inference.py

```diff
@@ -0,0 +1,27 @@
+from pathlib import Path
+
+import pytest
+
+from tests.examples.utils import copy_and_run_script, gen_cmd_fail_message, requires_gpu
+
+
+@pytest.fixture
+def example_dir() -> str:
+    return "examples/compressed_inference"
+
+
+@pytest.mark.example
+@requires_gpu
+class TestCompressedInference:
+    """
+    Tests for examples in the "compressed_inference" example folder.
+    """
+
+    def test_fp8_example_script(self, example_dir: str, tmp_path: Path):
```

Comment:
i think we need to annotate these so they don't run if the GPU doesn't support "fp8".

in general, all the tests need to be annotated, so they only run if the GPU supports the feature(s).

Reply:
I can take a look into this, but I don't think that is an issue with the current target runners (A100/H100).

Reply:
@dsikka Are you able to help shed light here on GPU compatibility concerns regarding the "fp8" example(s)?

Reply:
Per @dsikka, this is not a concern for this project.